### PR TITLE
Update LlamaCloud integration

### PR DESCRIPTION
**Pull request by Javtor**
# Description

Updates LlamaCloud integration

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [ ] No

## Type of Change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [x] Added new unit/integration tests
- [x] I stared at the code and made sure it makes sense

### CODE REVIEW

File: llama-index-core/llama_index/core/ingestion/pipeline.py [hunk 1]
```diff
@@ -336,163 +320,8 @@ def __init__(
...
-        assert (
-            pipeline_execution.id is not None
-        ), "Pipeline execution ID should not be None"
-
-        print(
-            f"Find your remote results here: {self.app_url}/"
-            f"pipelines/execution?id={pipeline_execution.id}"
-        )
-
-        return pipeline_execution.id
```
Comment (logan-markewich) [inline llama-index-core/llama_index/core/ingestion/pipeline.py:495]: Ah, so nice to remove this actually haha

File: llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py [hunk 2]
```diff
@@ -62,6 +86,110 @@ def __init__(
         self._service_context = None
         self._callback_manager = callback_manager or Settings.callback_manager
 
+    def _wait_for_pipeline_ingestion(
```
Comment (logan-markewich) [inline llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py:89]: Note to self: At some point, this probably needs to be async

File: llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py [hunk 3]
```diff
@@ -74,9 +202,10 @@ def from_documents(  # type: ignore
         app_url: Optional[str] = None,
         timeout: int = 60,
         verbose: bool = False,
+        raise_on_error: bool = False,
         **kwargs: Any,
     ) -> "LlamaCloudIndex":
-        """Build a Vectara index from a sequence of documents."""
+        """Build a LlamaCloud managed index from a sequence of documents."""
```
Comment (logan-markewich) [inline llama-index-integrations/indices/llama-index-indices-managed-llama-cloud/llama_index/indices/managed/llama_cloud/base.py:208]: lol

**Review by logan-markewich — APPROVED** [review]
Nice, this looks super good to me!

File: llama-index-core/llama_index/core/ingestion/api_utils.py [hunk 4]
```diff
@@ -1,151 +1,37 @@
...
-    return [
-        SentenceSplitter(),
-        OpenAIEmbedding(),
-    ]
 
 
 def get_client(
     api_key: Optional[str] = None,
     base_url: Optional[str] = None,
     app_url: Optional[str] = None,
```
Comment (sourabhdesai) [inline llama-index-core/llama_index/core/ingestion/api_utils.py:15]: what's `app_url` for? seems like its not being used?

**Review by sourabhdesai — APPROVED** [review]

File: llama-index-core/llama_index/core/ingestion/api_utils.py (hunk 4 above)
Comment (Disiok) [reply llama-index-core/llama_index/core/ingestion/api_utils.py:15]: previously, this was used to print link to go to the webapp.

**Comment by logan-markewich** [comment]
Besides some nits, I think this is good to merge. Don't want to hold up anything, so going to merge

**Pull request merged**